README.md

```diff
@@ -86,11 +86,24 @@ cd $GOPATH/src/github.com/optimizely/go-sdk
 go install
 ```
 
+NOTE:
+We practice trunk-based development, and as such our default branch, `master` might not always be the most stable. We do tag releases on Github and you can pin your installation to those particular release versions. One way to do this is to use *Go Modules* for managing external dependencies:
```

Comment:
Should we link to [blog.golang.org](https://blog.golang.org/using-go-modules) or the [github wiki](https://github.com/golang/go/wiki/Modules)?